src/Kafka/Producer.purs

```diff
@@ -148,22 +148,58 @@ type ProducerBatchImpl =
     , topicMessages :: Array TopicMessagesImpl
     )
 
+-- | see [Options](https://kafka.js.org/docs/producing#options)
+-- |
+-- | * `allowAutoTopicCreation`
+-- |   * Allow topic creation when querying metadata for non-existent topics
+-- |   * default: `true`
+-- | * `idempotent`
+-- |   * If enabled producer will ensure each message is written exactly once. Acks must be set to `-1` ("all"). Retries will default to `MAX_SAFE_INTEGER`.
+-- |   * default: `false`
+-- | * `maxInFlightRequests`
+-- |   * Max number of requests that may be in progress at any time. If falsey then no limit.
+-- |   * default: `null` (no limit)
+-- | * `metadataMaxAge`
+-- |   * The period of time in milliseconds after which we force a refresh of metadata even if we haven't seen any partition leadership changes to proactively discover any new brokers or partitions
+-- |   * default: `300000`
+-- | * `transactionTimeout`
+-- |   * The maximum amount of time in ms that the transaction coordinator will wait for a transaction status update from the producer before proactively aborting the ongoing transaction. If this value is larger than the `transaction.max.timeout.ms` setting in the broker, the request will fail with a `InvalidTransactionTimeout` error
+-- |   * default: `60000`
+-- | * `transactionalId`
+-- |   * The `transactionalId` allows Kafka to fence out zombie instances by rejecting writes from producers with the same `transactionalId`, allowing only writes from the most recently registered producer. To ensure EoS (Exactly-once Semantics) in a stream processing application, it is important that the `transactionalId` is always the same for a given input topic and partition in the read-process-write cycle.
+-- |   * see [Choosing a `transactionalId`](https://kafka.js.org/docs/transactions#choosing-a-transactionalid)
```

Comment:
Should we perhaps update the example values to PS ones? E.g. `null -> Nothing`, `-1 -> Kafka.Producer.AcksAll`, `300000 -> Data.Time.Duration.Milliseconds 300000.0`?